### PR TITLE
Rename event registration to presence tracing (DEV)

### PR DESCRIPTION
**Pull request by mtwalli**
- Renaming ☝️ 

Alt Text
### Testing
- Build device and deviceForTesters and go through the presence tracing screens

### CODE REVIEW

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/prsencetracing/storage/repo/DefaultTraceLocationRepositoryTest.kt [hunk 1]
```diff
@@ -1,9 +1,10 @@
-package de.rki.coronawarnapp.eventregistration.storage.repo
+package de.rki.coronawarnapp.prsencetracing.storage.repo
```
Comment (vaubaehn) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/prsencetracing/storage/repo/DefaultTraceLocationRepositoryTest.kt:1]: see above

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/prsencetracing/storage/retention/CheckInCleanerTest.kt [hunk 2]
```diff
@@ -1,7 +1,8 @@
-package de.rki.coronawarnapp.eventregistration.storage.retention
+package de.rki.coronawarnapp.prsencetracing.storage.retention
```
Comment (vaubaehn) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/prsencetracing/storage/retention/CheckInCleanerTest.kt:1]: see above

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/prsencetracing/storage/retention/TraceLocationCleanerTest.kt [hunk 3]
```diff
@@ -1,7 +1,8 @@
-package de.rki.coronawarnapp.eventregistration.storage.retention
+package de.rki.coronawarnapp.prsencetracing.storage.retention
```
Comment (vaubaehn) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/prsencetracing/storage/retention/TraceLocationCleanerTest.kt:1]: see above

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/prsencetracing/storage/retention/TraceLocationDbCleanUpPeriodicWorkerTest.kt [hunk 4]
```diff
@@ -1,8 +1,11 @@
-package de.rki.coronawarnapp.eventregistration.storage.retention
+package de.rki.coronawarnapp.prsencetracing.storage.retention
```
Comment (vaubaehn) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/prsencetracing/storage/retention/TraceLocationDbCleanUpPeriodicWorkerTest.kt:1]: see above

File: Corona-Warn-App/src/test/java/de/rki/coronawarnapp/prsencetracing/storage/retention/TraceLocationRetentionTest.kt [hunk 5]
```diff
@@ -1,6 +1,8 @@
-package de.rki.coronawarnapp.eventregistration.storage.retention
+package de.rki.coronawarnapp.prsencetracing.storage.retention
```
Comment (vaubaehn) [inline Corona-Warn-App/src/test/java/de/rki/coronawarnapp/prsencetracing/storage/retention/TraceLocationRetentionTest.kt:1]: see above

**Review by vaubaehn — COMMENTED** [review]
@mtwalli 
Hi Mohamed, your GIF expresses it all! 👍 
Looks like you replaced `eventregistration` with `presencetracing` with RegExp, and there was a typo in one term that used 'prsencetracing'? See following comments.
There was also inconsistent usage of `xyz.presencetracing.location(s)`, I assume it should be `locations` everywhere?

Is this PR to be expected to be the last before the release of 2.0?

Have a good night!

**Comment by mtwalli** [comment]
> @mtwalli
> Hi Mohamed, your GIF expresses it all! 👍
> Looks like you replaced `eventregistration` with `presencetracing` with RegExp, and there was a typo in one term that used 'prsencetracing'? See following comments.
> There was also inconsistent usage of `xyz.presencetracing.location(s)`, I assume it should be `locations` everywhere?
> 
> Is this PR to be expected to be the last before the release of 2.0?
> 
> Have a good night!

@vaubaehn Thank you very much for the great feedback! I will try to answer your questions here
- Yes, it should be the same everywhere. I made a typo while renaming the `test` package. (Fixed now)
- Yes , it should be `locations` not `location` , I renamed `events` -> `locations`
   I hope there is none now 
 
  Screenshot 2021-04-14 at 03 41 31
- Yes, that is is supposed be a " final" PR to handle the inconsistency in naming , unless we got some critical bugs to fix :)

I hope I was able to answer your questions.

**Review by d4rken — DISMISSED** [review]
lgtm

**Comment by harambasicluka** [comment]
We need to clarify if we get a new RC and if we want to put this in 2.0.1 or 2.1.0. So please do not merge!

**Comment by harambasicluka** [comment]
As this can't be tested as we need to we'll go the risk and ship it with 2.1.0 which than may cause a migration.

**Review by d4rken — APPROVED** [review]

**Review by LukasLechnerDev — APPROVED** [review]
LGTM! Did a short presencetracing smoke test and also tested it after upgrading from 2.0.0. Everything worked fine. 

Thanks for the clean up. 👍

**Comment by vaubaehn** [comment]
@mtwalli
> I hope I was able to answer your questions.

Yes, thank you very much :)

>     * Yes, that is is supposed be a " final" PR to handle the inconsistency in naming , unless we got some critical bugs to fix :)

I assumed, that your PR is somehow necessary for the testing of v2.0 - but as it's now timed for 2.1, I believe testing has already started, and unless there is no blocking bug, we can expect 2.0 soon,,,

**I'm so excited!**
I'm so excited!

You all did a great job, thanks to you all! ❤️

**Pull request merged**